task_management_app/urls.py

```diff
@@ -0,0 +1,37 @@
+from django.urls import path
+from .views import (
+    LoginForm,
+    SignupForm,
+    HomePage,
+    LogoutPage,
+    ProfileView,
+    TaskCreate,
+    TaskView,
+    TaskDetails,
+    CommentData,
+    DeleteTask,
+    UpdateTask,
+    CommentShow,
+    UserCreate,
+    UserList,
+    TaskSearch,
+)
+
+urlpatterns = [
+    path("", LoginForm.as_view(), name="loginform"),
+    path("signupform/", SignupForm.as_view(), name="signupform"),
+    path("homepage/", HomePage.as_view(), name="homepage"),
+    # path('userhome/',userhome.as_view(),name="userhome"),
+    path("logoutpage/", LogoutPage.as_view(), name="logoutpage"),
+    path("Profileview/", ProfileView.as_view(), name="profileview"),
+    path("TaskCreate/", TaskCreate.as_view(), name="TaskCreate"),
+    path("TaskView/", TaskView.as_view(), name="TaskView"),
+    path("Taskdetails/<int:id>", TaskDetails.as_view(), name="Taskdetails"),
```

Comment:
please do not use capital url names and try to improve urls like task-details or task/details/